docs/monitors/system-utilization.md

```diff
@@ -1,34 +1,44 @@
 <!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
 
-# signalfx-system-utilization
+# system-utilization
 
 
 This monitor reports utilization metrics for Windows
 Hosts.  It is used to drive the Windows Smart Agent host navigator view and dashboard
 content.
 
+Windows Performance Counters
```

Comment:
Formatting is off, either remove this line or make it a subheader with `##`.  You can use Markdown in the source Go comments.